python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -184,6 +184,7 @@ class LogisticRegression(Base):
         self.fit_intercept = fit_intercept
         self.max_iter = max_iter
         self.linesearch_max_iter = linesearch_max_iter
+        self.l1_ratio = None
```

Comment:
Stray change? Or needed for pickling?

Reply:
Needed for pickling, and makes pickling the unfit model a little more robust/consistent